client/src/main/java/io/pravega/client/stream/ReaderGroup.java

```diff
@@ -27,6 +27,13 @@
 public interface ReaderGroup {
 
     /**
+     * Returns metrics for this reader group.
+     * 
+     * @return a ReaderGroupMetrics object for this reader group.
+     */
+    ReaderGroupMetrics getMetrics();
```

Comment:
`getMetrics` currently returns an object that allows the application to call other methods to get the metrics, currently just one. This level of indirection seems to be unnecessary, especially because the reader group is implementing the interface and return `this`. Are you trying to be future-proof?